storyblok/storyblok-design-system
# refactor:  sbCard to template syntax

## Pull request type

Jira Link: - 




## How to test this PR


Compare: 
* https://blok.ink/?path=/story/design-system-components-sbcard--default
* https://storyblok-design-system-git-refactor-ar-12-card-storyblok-com.vercel.app/?path=/story/design-system-components-sbcard--default

It's also good to check in the app:
`cd storyblok-design-system && git checkout refactor/ar-12-card`
Then start the server of the app

## What is the new behavior?



- it uses the template syntax
- 
- 

## Other information
